cmd/icingadb/main.go

```diff
@@ -379,9 +392,11 @@ func monitorRedisSchema(logger *logging.Logger, rc *icingaredis.Client, pos stri
 // checkRedisSchema verifies rc's icinga:schema version.
 func checkRedisSchema(logger *logging.Logger, rc *icingaredis.Client, pos string) (newPos string, err error) {
 	if pos == "0-0" {
-		defer time.AfterFunc(3*time.Second, func() { logger.Info("Waiting for current Redis schema version") }).Stop()
+		defer time.AfterFunc(3*time.Second, func() {
+			logger.Info("Waiting for Icinga 2 to write into Redis. Please make sure you have started Icinga 2 and Icinga DB feature is enabled")
```

Comment:
"[...] and **the** Icinga DB feature [...]"

Additionally, I'd either end both sentences with a period here, or just do the same ", please" as with the other messages.